business/definitionService.js

```diff
@@ -57,8 +57,11 @@ class DefinitionService {
       return this.compute(coordinates, curation)
     }
     const existing = force ? null : await this.definitionStore.get(coordinates)
-    const result =
-      get(existing, '_meta.schemaVersion') === currentSchema ? existing : await this.computeAndStore(coordinates)
+    let result
+    if (get(existing, '_meta.schemaVersion') === currentSchema) {
+      this.logger.info('current definition exists', { coordinates: coordinates.toString() })
```

Comment:
perhaps the following three messages to capture all the states:

* "computed definition available"
* "recomputed definition available"
* "definition not available"

Question, do you think we'll ever care about the difference between the first time we compute a definition (e.g., `existing === null`) and the N times we recompute? And for recompute, how often it is becuase the schema was out of date vs force === true?

Reply:
force=true should only ever be in local dev.

Maybe we could start to track when definitions are invalidated?